test/tools/ossfuzz/abiV2Fuzzer.cpp

```diff
@@ -0,0 +1,78 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <fstream>
+#include <test/tools/evmoneRunner.h>
+#include "abiV2FuzzerCommon.h"
+
+using namespace dev::test::abiv2fuzzer;
+using namespace external::evmone;
+using namespace std;
+using namespace dev;
+
+extern "C" int LLVMFuzzerTestOneInput(uint8_t const* _data, size_t _size)
```

Comment:
Is this the main entry function? If yes, why does it not use the ProtoConverter (or did I overlook something)?